client/src/main/scala/org/http4s/client/oauth1.scala

```diff
@@ -0,0 +1,102 @@
+package org.http4s
+package client
+
+import java.nio.charset.StandardCharsets
+
+import org.http4s.util.{UrlFormCodec, UrlCodingUtils}
+import org.http4s.headers.Authorization
+import org.http4s.util.string._
+
+import javax.crypto
+
+import scala.collection.mutable.ListBuffer
+import scalaz.concurrent.Task
+
+/** Basic OAuth1 message signing support
+  * 
+  * This feature is not considered stable.
+  */
+object oauth1 {
```

Comment:
I'm inclined to think this is a package object, at which point Consumer and Token move out into the package.  But I don't think it's going to make a grand difference either way.